src/components/confirm/confirm.component.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Dialog from '../dialog/dialog.component';
+import Button from '../button';
+import StyledConfirmButtons from './confirm.style';
+
+class Confirm extends Dialog {
+  /**
+   * Returns main classes for the component combined with
+   * dialog main classes.
+   */
+  get mainClasses() {
+    return classNames(super.mainClasses);
```

Comment:
I'm not sure we need these class names any more.

Reply:
The scope of the ticket was to change the styling only. I think that all of these classnames will be removed when the component itself is rewritten.

Reply:
In general I think it's safe to remove any code that the addition of styled components will replace, whilst avoiding rewriting the implementation. However, as this is an inherited component I'd be inclined to leave it for the future ticket